imports/ui/lists/trendingAgents.jade

```diff
@@ -0,0 +1,21 @@
+template(name="trendingAgents")
+  h4.col-md-5 Infectious Agents trending in the last:
+  select#trendingRange.col-md-6.form-control
+    option(value="year") Year
+    option(value="month") Month
+    option(value="week") Week
+  table#trendingAgentsTable.table.table-striped
+    tbody
+      each trendingAgents
```

Comment:
I think the sort order from the query might not be preserved here because I'm seeing some "trending at" scores that are out of order.